src/server/server.ts

```diff
@@ -75,25 +155,24 @@ export class GraphQLTransportWSServer {
         return;
       }
 
+      const connectionContext: ConnectionContext = Object.create(null);
+      connectionContext.isLegacy = false;
+      connectionContext.socket = socket;
+
       // Regular keep alive messages if keepAlive is set
       if (keepAlive) {
```

Comment:
we need to make sure in client that keepAlive is optional
(maybe start monitor only after the first one is recieved?)